jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -68,10 +68,10 @@ blockchain_configuration:
     per_certificate_fees:
       # (optional) if not specified, the pool registration certificate fee will be
       # the one set by linear_fees.certificate
-      certificate_pool_registration: 5
+      certificate_pool_registration: 150000000
```

Comment:
I don't understand why you needed to change this value?

Reply:
This is consistent with the expected charge on the mainnet (actually about 1/3-2/3 of the charge, which will be 250-500 Ada).  It's reasonable to set this slightly lower on the testnet.

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.